pkg/api/api.go

```diff
@@ -242,7 +242,7 @@ func (a *API) RegisterDistributor(d *distributor.Distributor, pushConfig distrib
 	wrappedDistributor := a.cfg.wrapDistributorPush(d)
 
 	a.RegisterRoute("/api/v1/push", push.Handler(pushConfig.MaxRecvMsgSize, a.sourceIPs, a.cfg.SkipLabelNameValidationHeader, wrappedDistributor), true, false, "POST")
-	a.RegisterRoute("/api/v1/push/otlp/v1/metrics", push.OLTPHandler(pushConfig.MaxRecvMsgSize, a.sourceIPs, a.cfg.SkipLabelNameValidationHeader, wrappedDistributor), true, false, "POST")
+	a.RegisterRoute("/otlp/v1/metrics", push.OTLPHandler(pushConfig.MaxRecvMsgSize, a.sourceIPs, a.cfg.SkipLabelNameValidationHeader, wrappedDistributor), true, false, "POST")
```

Comment:
Please edit the CHANGELOG entry and the client used in integration tests `integration/e2emimir/client.go`